README.md

```diff
@@ -1,67 +1,42 @@
 ﻿# .NET Core Home
 
-The core repository is the starting point to engage in and learn about the
-.NET Core stack. 
-
-> **Note:** please note that this repository is not for filing product issues.
-> If you run into an issue using .NET Core, there are multiple repos where you can 
-> file an issue:
-> * [dotnet/cli](https://github.com/dotnet/cli) - for CLI tools and questions
-> * [dotnet/corefx](https://github.com/dotnet/corefx) - for API issues and questions
-> * [dotnet/coreclr](https://github.com/dotnet/coreclr) - for runtime issues
```

Comment:
Should we add core-setup? That's where we build netcoreapp, the host, and the runtime installers.